music21/duration.py

```diff
@@ -681,6 +697,29 @@ def quarterConversion(qLen: OffsetQLIn) -> QuarterLengthConversion:
     # remove the largest type out there and keep going.
 
     qLenRemainder = opFrac(qLen - typeToDuration[closestSmallerType])
+
+    # one opportunity to define a tuplet if remainder can be expressed as one
+    # by expressing the largest type (components[0]) in terms of the same tuplet
+    if isinstance(qLenRemainder, fractions.Fraction):
```

Comment:
I see the limitation -- my attempt is just picking up after "remove the largest type out there and keep going", but in the case of 11/3 (3.666 QL) we need to start with a component of an eighth, not a half, so we can do 11 x eighth-triplet.

Maybe this needs to be extracted into a subroutine that can run during the "try eight times" branch below.

Reply:
Thanks @gregchapman-dev for finding it!

Reply:
@gregchapman-dev game to take another look?

11/3 now gives this (and the consolidation back into quarter rests for the first three beats is then the task for #904...)
<img width="892" alt="Screen Shot 2022-03-01 at 8 40 34 PM" src="https://user-images.githubusercontent.com/38668450/156278441-fccef907-d427-4226-9e81-ab5799a44e90.png">